contrib/test_jenkins.sh

```diff
@@ -29,6 +29,11 @@ ucx_inst=${WORKSPACE}/install
 CUDA_MODULE="dev/cuda11.1.1"
 GDRCOPY_MODULE="dev/gdrcopy2.1_cuda11.1.1"
 
+if [[ "$RUNNING_IN_AZURE" == "yes" ]]; then
+	realdir=$(realpath $(dirname $0))
+	source ${realdir}/../buildlib/az-helpers.sh
```

Comment:
currently, whoever runs this script is responsible to source az-helpers before
see https://github.com/openucx/ucx/blob/4967de66bdab506c32e788574729cc3ec944c845/buildlib/tests.yml#L28